### DISCUSSION TITLE
Common research model (CRM) wing tutorial solution diverge

### DISCUSSION THREAD

**Discussion by Ryupuriwatt** [mdolab/dafoam#553] [Tutorials]
![image](https://github.com/mdolab/dafoam/assets/115110434/b4aaac85-4c8b-46e7-8370-ec9acb27ec7c)
I would like to know why the primal solver diverge. In this case, I just run the tutorial without modify anything.

**Comment by Ryupuriwatt** [comment]
What I did is not same as the tutorial is just only I use 12 cores for calculation.

**Comment by friedenhe** [comment]
Thanks for the feedback. We reproduced the issue and will fix it soon.

**Reply by Ryupuriwatt** [reply]
Thank you very much profressor he.

**Comment by friedenhe** [comment]
It seems that the problem was caused by a bug in the createPatchDict file. It somehow set the sym and inout patches in a wrong way. We just fixed this issue (https://github.com/DAFoam/tutorials/commit/6feea1200bfca65652c936111304457fa19cebd6) and the CRM case should be running fine now.

**Reply by Ryupuriwatt** [reply]
![image](https://github.com/mdolab/dafoam/assets/115110434/fc61e03d-b3e0-454e-8c1b-3da35b818a89)
After professor fixed the bug the primal solver still diverge.

**Reply by Ryupuriwatt** [reply]
If professor have any idea about this error please inform me.

**Comment by friedenhe** [comment]
@Ryupuriwatt, we just ran the latest tutorial for the CRM case using DAFoam v3.1.0, and both v3 and v2 scripts work. We did not see the convergence issue you had.... BTW:  we did fix a bug in the v3 script (the aoa's normal direction was wrong). But this bug fix won't impact the flow convergence..
